lib/screens/createaccount_screen.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/material.dart';
+import 'package:intern_task/util/authentication_service.dart';
+import 'package:intern_task/util/widget/label_checkbox.dart';
+
+class CreateAccountScreen extends StatefulWidget {
+  final VoidCallback showLoginScreen;
+  const CreateAccountScreen({super.key, required this.showLoginScreen});
+
+  @override
+  State<CreateAccountScreen> createState() => _CreateAccountScreenState();
+}
+
+class _CreateAccountScreenState extends State<CreateAccountScreen> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  final _emailController = TextEditingController();
+  final _passwordController = TextEditingController();
+  final _confirmPasswordController = TextEditingController();
+  bool _hidePassword = true;
+  bool _hideConfirmPassword = true;
+
+  @override
+  void dispose() {
+    _emailController.dispose();
+    _passwordController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(title: const Text('Sign Up')),
+        body: Center(
+          child: Container(
```

Comment:
this Center, Container, Padding is unnecessary here.
use cross axis alignment in column